hikari/audit_logs.py

```diff
@@ -260,7 +332,29 @@ class MemberMoveEntryInfo(MemberDisconnectEntryInfo):
     """Extra information for the voice chat based member move entry."""
 
     channel_id: snowflakes.Snowflake = attr.ib(repr=True)
-    """The amount of members who were disconnected from voice in this entry."""
+    """The channel that the member(s) have been moved to"""
+
+    async def fetch_channel(self) -> GuildVoiceChannel:
+        """Fetch the guild text based channel where these message(s) were deleted.
+
+        Returns
+        -------
+        hikari.channels.GuildVoiceChannel
+
+        Raises
+        ------
+        hikari.errors.UnauthorizedError
+            If you are unauthorized to make the request (invalid/missing token).
+        hikari.errors.ForbiddenError
+            If you are missing the `READ_MESSAGES` permission in the channel.
+        hikari.errors.NotFoundError
+            If the channel is not found.
+        hikari.errors.InternalServerError
```

Comment:
`RateLimitTooLongError` and  `RateLimitedError` missing here

Reply:
right